github-graphql/README.md

```diff
@@ -0,0 +1,30 @@
+# How to use GraphQL with Rust
```

Comment:
I will note down is a separate issue to make a little refactor of the GraphQL code. It has grown a bit organically but now I think we can establish some guidelines on how to do that